src/reducers/org.js

```diff
@@ -518,6 +518,28 @@ const refileSubtree = (state, action) => {
   return state;
 };
 
+const addNote = (state, action) => {
+  // See org-add-note (C-c C-z) and variable org-log-note-headings
+  const { noteText } = action;
+  console.log(noteText);
+
+  const headerId = state.get('selectedHeaderId');
+  const headers = state.get('headers');
+  const headerIndex = indexOfHeaderWithId(headers, headerId);
+  return state.updateIn(['headers', headerIndex], (header) =>
+    // TODO: Don't do it like this, because this overwrites the
+    //       complete body of the header.
+    // TODO: I don't think it works like this. Probably it's a good
+    //       idea to extend `_updateHeaderFromDescription` from
+    //       lib/parse_org.js.
+    // It's not enough to set rawDescription; the whole header must be
+    // reconstructed. Either parse modified rawDescription or directy
+    // update description (which is parsed).
+
+    header.update('rawDescription', (rawDescription) => `${noteText}\n${rawDescription}`)
```

Comment:
I still have to think about how to best integrate the note in the header... it's not as trivial as I initially thought ;)

Reply:
I think I'll go with prepending/inserting to `rawDescription` and then parse the whole thing again.

I have to check, but I think the notes are better seen as part of the description, not as one of the "prefix elements"... because it should end up as "parsed description".

Reply:
> I still have to think about how to best integrate the note in the header... it's not as trivial as I initially thought ;)

It's definitively not trivial. It'll have to be included in the domain model that we have for a header. I've documented the way I'd do it here: https://github.com/200ok-ch/organice/pull/423/files#diff-d675f8aab42787975acbd1fe26257689R530